### DISCUSSION TITLE
并行时间

### DISCUSSION THREAD

**Discussion by 775019309** [deepmodeling/deepmd-kit#37] [Q&A]
Hello! I have set the num_inter_threads and num_intra_threads，but I haven't seen a big improvement in training time.A node has 24 CPUs, but only one CPU is used.I hope you can help me.

**Comment by njzjz** [comment]
Which parameter do you set? `num_inter_threads*num_intra_threads=num_total_threads`

TensorFlow has explained it in the comment:
https://github.com/tensorflow/tensorflow/blob/r1.12/tensorflow/core/protobuf/config.proto#L297-L310

Here is also an explanation:
https://stackoverflow.com/questions/41233635/meaning-of-inter-op-parallelism-threads-and-intra-op-parallelism-threads

**Comment by 775019309** [comment]
I have tried many kinds of inter and intra parameters,sush as 0 and 0 or 1 and 2 or 2 and 1......but it still used one CPU.My management system is PBS,can you provide a script for submitting task?Thank you very much!

**Comment by 775019309** [comment]
I have solved this problem.Thanks a lot,and I want to know if the test datas also can be parallel calculated?

**Comment by xjhsafeguards** [comment]
Do you mind if I ask how did you solve the problem? Is there any specific set up for those parameters?
